README.md

```diff
@@ -273,6 +273,14 @@ request.get('http://some.server.com/', {
     'bearer': 'bearerToken'
   }
 });
+// or with SPNEGO/Kerberos
+request.get('http://some.server.com/').auth(null, null, false, null, true);
+// or
+request.get('http://some.server.com/', {
+  'auth': {
+    'negotiate': true
```

Comment:
As far as I understand there are other authenticating mechanisms such as [NTLM](https://en.wikipedia.org/wiki/NT_LAN_Manager) for example, so `negotiate: true` might not be the most approprite configuration to use. Will investigate further these days.